redisinsight/ui/src/pages/rdi/connection-form/ConnectionForm.tsx

```diff
@@ -0,0 +1,213 @@
+        <Form>
+          <EuiForm component="div" data-testid="connection-form">
+            <EuiSplitPanel.Outer className={styles.connectionFormPanel} borderRadius="none">
+              <EuiSplitPanel.Inner>
+                <EuiTitle size="s">
+                  <h3>Connect to RDI</h3>
+                </EuiTitle>
+                <EuiSpacer />
+                <EuiFormRow label="RDI Alias" fullWidth>
+                  <Field name="name">
+                    {({ field }: { field: FieldInputProps<string> }) => (
+                      <EuiFieldText
+                        data-testid="connection-form-name-input"
+                        fullWidth
+                        placeholder="Enter RDI Alias"
+                        maxLength={500}
+                        {...field}
+                      />
+                    )}
+                  </Field>
+                </EuiFormRow>
+                <EuiFormRow label="URL*" fullWidth>
+                  <Field name="url">
+                    {({ field }: { field: FieldInputProps<string> }) => (
+                      <EuiFieldText
+                        data-testid="connection-form-url-input"
+                        fullWidth
+                        placeholder="Enter URL"
+                        disabled={!!editInstance}
+                        append={!editInstance ? <UrlTooltip /> : undefined}
+                        {...field}
+                      />
+                    )}
+                  </Field>
+                </EuiFormRow>
+                <EuiFormRow label="Username*" fullWidth>
+                  <Field name="username">
+                    {({ field }: { field: FieldInputProps<string> }) => (
+                      <EuiFieldText
+                        data-testid="connection-form-username-input"
+                        fullWidth
+                        placeholder="Enter Username"
+                        maxLength={500}
+                        {...field}
+                      />
+                    )}
+                  </Field>
+                </EuiFormRow>
+                <EuiFormRow label="Password*" fullWidth>
+                  <Field name="password">
+                    {({ field }: { field: FieldInputProps<string> }) => (
+                      <EuiFieldPassword
+                        data-testid="connection-form-password-input"
+                        className={styles.passwordField}
+                        fullWidth
+                        placeholder="Enter Password"
+                        maxLength={500}
+                        {...field}
+                      />
+                    )}
+                  </Field>
+                </EuiFormRow>
+              </EuiSplitPanel.Inner>
+              <EuiSplitPanel.Inner grow={false}>
+                <EuiFlexGroup justifyContent="spaceBetween">
+                  <EuiFlexItem grow={false}>
+                    <ValidationTooltip isValid={isValid} errors={errors}>
```

Comment:
Is it possible to create component SubmitButton? like in ManualConnectionForm, but move it outside and use it everywhere?

Reply:
Could you explain the reasoning behind this request? The button itself doesn't have much logic and isn't used outside of the `ConnectionForm` component so reuse doesn't seem likely.

Reply:
I'm guess that we can move button with tooltip in one component and reuse it (in database connection form and in rdi instance form) But now I'm see that is not so easy, so we can go with ValidationTooltip

Reply:
Oh, you were also referring to the Tooltip since they are the same across forms. If you feel strongly about this request I can create a shared tooltip that both forms can use.